src/menus/menu.jsx

```diff
@@ -206,7 +206,7 @@ const Menu = React.createClass({
     rootStyle.transition = Transitions.easeOut('250ms', ['opacity', 'transform']);
     rootStyle.transform = 'translate3d(0,-8px,0)';
     rootStyle.opacity = 0;
-    rootStyle = AutoPrefix.all(rootStyle);
+    rootStyle = autoPrefix.all(rootStyle, this.state.muiTheme);
```

Comment:
Turns out, this is dead code, `componentWillLeave' is not used in our code base. The `<Menu />` is always wrapped inside a `Paper` or a `Popover` not inside a `ReactTransitionGroup`. Plus, that sounds like a bad pattern.